### PR TITLE
Bench for bus stops

### PR DESCRIPTION
**Pull request by matkoniecz**
I am bit dubious about usefulness of this quest but it is asked after all reasonable quests were asked in a given region and I want to reduce number of open issues to 100 :)

### CODE REVIEW

File: app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+class AddBenchStatusOnBusStop(o: OverpassMapDataAndGeometryApi, r: ResurveyIntervalsStore) : SimpleOverpassQuestType<Boolean>(o) {
+
+    override val tagFilters = """
+        nodes with
+        (
+          (public_transport = platform and ~bus|trolleybus|tram ~ yes)
+          or
+          (highway = bus_stop and public_transport != stop_position)
+        )
+        and physically_present != no and naptan:BusStopType != HAR
```
Comment (peternewman) [inline app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt:19]: Ah now I see why you were asking in the other PR. Is it worth pulling this out so it's a common rule across these two bus stop quests and any future ones. Or at least adding comments to both saying to update the other one if it changes.

Or actually TBH all of this filter apart from the bench bit could be common couldn't it?

File: app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        and (!bench or bench older today -${r * 5} years)
+    """
+
+    override val commitMessage = "Add whether bus stop has a bench"
```
Comment (peternewman) [inline app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt:23]:
```suggestion
    override val commitMessage = "Add whether a bus stop has a bench"
```

**Review by peternewman — COMMENTED** [review]
A few comments for what they're worth.

File: app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    override val tagFilters = """
+        nodes with
+        (
+          (public_transport = platform and ~bus|trolleybus|tram ~ yes)
+          or
+          (highway = bus_stop and public_transport != stop_position)
+        )
+        and physically_present != no and naptan:BusStopType != HAR
+        and (!bench or bench older today -${r * 5} years)
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt:20]:
```suggestion
        and (!bench or bench older today -${r * 4} years)
```

(All these are using 2, 4, 8,... unless there is no good reason for an exception, let's stay with this IMO)

**Review by westnordost — APPROVED** [review]

**Comment by HolgerJeromin** [comment]
Ref #1079

**Comment by HolgerJeromin** [comment]
>  I want to reduce number of open issues to 100 :)

91! :-)

**Pull request merged**